gateway/dapiimpl/server_v1/dataapi_touch.go

```diff
@@ -25,7 +25,7 @@ func (s *DataApiServer) TouchDocument(
 
 	var newExpiry uint32
 	if in.Body.Expiry != nil {
-		expiry, errSt := httpTimeToGocbcorexExpiry(*in.Body.Expiry)
+		expiry, errSt := isoTimeToGocbcorexExpiry(*in.Body.Expiry)
```

Comment:
Does anywhere else need updating too?

Reply:
I had a look and I don't think so since all the other places `httpTimeToGocbcorexExpiry` is used is when extracting the expiry from the HTTP headers.